### PR TITLE
feat: optional namespaces

### PR DESCRIPTION
**Pull request by ganchoradkov**

### CODE REVIEW

File: wallets/react-wallet-v2/src/views/SessionProposalModal.tsx [hunk 1]
```diff
@@ -171,17 +204,32 @@ export default function SessionProposalModal() {
...
-              <Text h4 css={{ marginBottom: '$5' }}>{`Review ${chain} permissions`}</Text>
+              <Text css={{ marginBottom: '$5' }}>{`Review ${chain} permissions`}</Text>
               <SessionProposalChainCard requiredNamespace={requiredNamespaces[chain]} />
-              {renderAccountSelection(chain)}
+              {renderAccountSelection(`required:${chain}`)}
               <Divider y={2} />
             </Fragment>
           )
         })}
+        {optionalNamespaces && Object.keys(optionalNamespaces).length != 0 ? (
```
Comment (bkrem) [inline wallets/react-wallet-v2/src/views/SessionProposalModal.tsx:218]:
```suggestion
        {optionalNamespaces && Object.keys(optionalNamespaces).length !== 0 ? (
```

**Review by bkrem — APPROVED** [review]
LGTM, just some minor things 🚀 

Thanks for putting together the stripped down sample 💯  obviously a lot of components etc recreated there so let me know if I should review something specific that's different there 👍

**Comment by bkrem** [comment]
What actually happened here? Did Ilja do a drive-by on this PR? 😂 

If we want to revive this, @ganchoradkov maybe you can just force push from your local to reset or pull and reset to where the PR originally was (`git reset HEAD^^ --hard`)

**Comment by 0xAsimetriq** [comment]
> What actually happened here? Did Ilja do a drive-by on this PR? 😂
> 
> If we want to revive this, @ganchoradkov maybe you can just force push from your local to reset or pull and reset to where the PR originally was (`git reset HEAD^^ --hard`)

I managed to mess up yarn.lock file here, but all of Gancho's changes are in the eip155 wallet example atm.

**Pull request closed without merging**